drivers/sensor/stm32_temp/stm32_temp.c

```diff
@@ -143,7 +144,8 @@ static const struct stm32_temp_config stm32_temp_dev_config = {
 	.cal1_temp = DT_INST_PROP(0, ts_cal1_temp),
 	.cal2_temp = DT_INST_PROP(0, ts_cal2_temp),
 	.cal_vrefanalog = DT_INST_PROP(0, ts_cal_vrefanalog),
-	.cal_offset = DT_INST_PROP(0, ts_cal_offset)
+	.cal_offset = DT_INST_PROP(0, ts_cal_offset),
+	.ts_cal_resolution = (DT_INST_PROP(0, ts_cal_resolution) - 12)
```

Comment:
After doing the substraction `.ts_cal_resolution` should then be called `ts_cal_shift` as it's not a resolution anymore

```suggestion
	.ts_cal_shift = (DT_INST_PROP(0, ts_cal_resolution) - CAL_RES)
```
with
```
#define CAL_RES 12
```
(Prefer use of defines with explicit names rather than "magic"  numbers).
@ABOSTM, @FRASTM do you have an idea for a better name than `CAL_RES`

Reply:
RAW_DATA_RESOLUTION ?